packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -127,8 +128,9 @@ export const ProjectPendingLink = () => (
     Approval in progress
   </OutlineLink>
 );
+
 export const ProjectAllowedLink = ({ url }: LinkProps) => (
-  <BaseLink to={url}>View project</BaseLink>
+  <BaseLink to={url} target={`${useLandingPage(url)}`}>View project</BaseLink>
```

Comment:
One thing to remember is that these cards and links will be reused by the Projects modal later on, and we don't want them to open in a new tab from there. Ideally the `ProjectAllowedLink` would take a param that flags whether it should open in a new tab or not, and apply `target='_blank'` when that condition is met